### ISSUE TITLE
Unsafe unwrap()s in the frontend das api

### ISSUE THREAD

**Issue by linuskendall** [metaplex-foundation/digital-asset-rpc-infrastructure#62]
There are some unsafe unwraps in the frontend api - looks like we hit some particualr call but didn't have proper backtrace enabled.

**Comment by linuskendall** [comment]
```
Apr 21 09:30:40 dub18 das_api[1502600]: thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /opt/ghrunner/actions-runner/_work/das-api/das-api/digital-asset-rpc-infrastructure/digital_asset_types/src/dapi/common/asset.rs:167:54
```

**Comment by Lusitaniae** [comment]
```
das_api[3163395]: thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /opt/ghrunner/actions-runner/_work/das-api/das-api/digital-asset-rpc-infrastructure/digital_asset_types/src/dapi/common/asset.rs:167:54
das_api[3163395]: stack backtrace:
das_api[3163395]:    0: rust_begin_unwind
das_api[3163395]:              at ./rustc/1716932743a7b3705cbf0c34db0c4e070ed1930d/library/std/src/panicking.rs:579:5
das_api[3163395]:    1: core::panicking::panic_fmt
das_api[3163395]:              at ./rustc/1716932743a7b3705cbf0c34db0c4e070ed1930d/library/core/src/panicking.rs:64:14
das_api[3163395]:    2: core::panicking::panic
das_api[3163395]:              at ./rustc/1716932743a7b3705cbf0c34db0c4e070ed1930d/library/core/src/panicking.rs:114:5
das_api[3163395]:    3: digital_asset_types::dapi::common::asset::v1_content_from_json
das_api[3163395]:    4: digital_asset_types::dapi::common::asset::asset_to_rpc
das_api[3163395]:    5: <das_api::api::api_impl::DasApi as das_api::api::ApiContract>::get_asset::{{closure}}
das_api[3163395]:    6: jsonrpsee_core::server::rpc_module::RpcModule<Context>::register_async_method::{{closure}}::{{closure}}
das_api[3163395]:    7: jsonrpsee_server::transport::http::process_validated_request::{{closure}}
das_api[3163395]:    8: <futures_util::future::future::map::Map<Fut,F> as core::future::future::Future>::poll
das_api[3163395]:    9: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
das_api[3163395]:   10: <jsonrpsee_server::middleware::proxy_get_request::ProxyGetRequest<S> as tower_service::Service<http::request::Request<hyper::body::body::Body>>>::call::{{closure}}
das_api[3163395]:   11: <tower_http::cors::ResponseFuture<F> as core::future::future::Future>::poll
das_api[3163395]:   12: hyper::proto::h1::dispatch::Dispatcher<D,Bs,I,T>::poll_catch
das_api[3163395]:   13: <hyper::server::conn::upgrades::UpgradeableConnection<I,S,E> as core::future::future::Future>::poll
das_api[3163395]:   14: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
das_api[3163395]:   15: <tracing::instrument::Instrumented<T> as core::future::future::Future>::poll
das_api[3163395]:   16: jsonrpsee_server::future::FutureDriver<F>::drive
das_api[3163395]:   17: jsonrpsee_server::server::Server<B,L>::start_inner::{{closure}}
das_api[3163395]:   18: tokio::runtime::task::core::Core<T,S>::poll
das_api[3163395]:   19: tokio::runtime::task::harness::Harness<T,S>::poll
das_api[3163395]:   20: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
das_api[3163395]:   21: tokio::runtime::scheduler::multi_thread::worker::Context::run
das_api[3163395]:   22: tokio::macros::scoped_tls::ScopedKey<T>::set
das_api[3163395]:   23: tokio::runtime::scheduler::multi_thread::worker::run
das_api[3163395]:   24: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
das_api[3163395]:   25: tokio::runtime::task::core::Core<T,S>::poll
das_api[3163395]:   26: tokio::runtime::task::harness::Harness<T,S>::poll
das_api[3163395]:   27: tokio::runtime::blocking::pool::Inner::run
das_api[3163395]: note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

```

**Comment by gritsly** [comment]
Problem lies here:
https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/blob/main/digital_asset_types/src/dapi/common/asset.rs#L167

**Comment by dancamarg0** [comment]
Important to point out this don't seem to crash the main process and won't trigger systemd to restart the service because of that apparently.

We had to log in dub18 and restart das_api service twice in the last 24h to restore the service.

**Comment by linuskendall** [comment]
Most of these have been fixed, will be submitting a PR for the same.

**Comment by linuskendall** [comment]
Fixed by #75.
